Mage.Sets/src/mage/cards/m/MarwynsKindred.java

```diff
@@ -0,0 +1,67 @@
+package mage.cards.m;
+
+import mage.abilities.Ability;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.common.ConjureCardEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Outcome;
+import mage.constants.Zone;
+import mage.game.Game;
+import mage.util.CardUtil;
+import java.util.UUID;
+
+/**
+ * @author karapuzz14
+ */
+public final class MarwynsKindred extends CardImpl {
+
+
+    public MarwynsKindred(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.SORCERY}, "{X}{2}{G}{G}");
+
+        // Conjure a card named Marwyn, the Nurturer and X cards named Llanowar Elves onto the battlefield.
+        this.getSpellAbility().addEffect(new MarwynsKindredEffect());
+    }
+
+    private MarwynsKindred(final mage.cards.m.MarwynsKindred card) {
```

Comment:
no need for the `mage.cards.m.` here or anywhere else besides the package declaration, it's in namespace without qualifiers